api/routes/comments/comments.service.js

```diff
@@ -0,0 +1,23 @@
+const { dynamodb, TableName } = require("../../services/db");
+
+exports.createComment = async (postId, commentId, params) => {
+  const newComment = {
+    TableName,
+    Item: {
+      PK: `POST#${postId}`,
+      SK: `COMMENT#${commentId}`,
+      text: params.text,
```

Comment:
should be params.comment, not params.text

Reply:
👍